src/storage/cuckoo/cuckoo.c

```diff
@@ -296,14 +293,14 @@ cuckoo_setup(cuckoo_options_st *options, cuckoo_metrics_st *metrics)
     }
     ds = datapool_addr(pool);
 
-#if CC_ITT
-    cuckoo_malloc = __itt_heap_function_create("cuckoo_malloc", "pelikan");
-    for(size_t n = 0; n < max_nitem; ++n) {
-        __itt_heap_allocate_begin(cuckoo_malloc, item_size, 0);
-        void *it = OFFSET2ITEM(n);
-        __itt_heap_allocate_end(cuckoo_malloc, &it, item_size, 0);
+    if (cuckoo_instrument) {
```

Comment:
is there a variable that says "Running under VTune"? If yes, replace this with it, otherwise just do it unconditionally.